src/test/ui/not-enough-arguments.stderr

```diff
@@ -1,14 +1,44 @@
 error[E0061]: this function takes 4 arguments but 3 arguments were supplied
-  --> $DIR/not-enough-arguments.rs:10:3
+  --> $DIR/not-enough-arguments.rs:26:3
    |
-LL | fn foo(a: isize, b: isize, c: isize, d:isize) {
-   | --------------------------------------------- defined here
-...
 LL |   foo(1, 2, 3);
    |   ^^^ -  -  - supplied 3 arguments
    |   |
    |   expected 4 arguments
+   |
+note: function defined here
+  --> $DIR/not-enough-arguments.rs:5:1
+   |
+LL | fn foo(a: isize, b: isize, c: isize, d:isize) {
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error[E0061]: this function takes 6 arguments but 1 argument was supplied
+  --> $DIR/not-enough-arguments.rs:28:3
+   |
+LL |   bar(1);
+   |   ^^^ - supplied 1 argument
+   |   |
+   |   expected 6 arguments
+   |
+note: function defined here
+  --> $DIR/not-enough-arguments.rs:9:1
+   |
+LL | / fn bar(
```

Comment:
Can we instead point at the ident instead of the full fn head?

Reply:
```
 supplied
  --> $DIR/not-enough-arguments.rs:26:3
   |
LL |   foo(1, 2, 3);
   |   ^^^ -  -  - supplied 3 arguments
   |   |
   |   expected 4 arguments
   |
note: function defined here
  --> $DIR/not-enough-arguments.rs:5:4
   |
LL | fn foo(a: isize, b: isize, c: isize, d:isize) {
   |    ^^^ ^^^^^^^^^ ^^^^^^^^^ ^^^^^^^^^ ^^^^^^^^

error[E0061]: this function takes 6 arguments but 1 argument was supplied
  --> $DIR/not-enough-arguments.rs:28:3
   |
LL |   bar(1);
   |   ^^^ - supplied 1 argument
   |   |
   |   expected 6 arguments
   |
note: function defined here
  --> $DIR/not-enough-arguments.rs:9:4
   |
LL | fn bar(
   |    ^^^
LL |     a: i32,
   |     ^^^^^^^
LL |     b: i32,
   |     ^^^^^^^
LL |     c: i32,
   |     ^^^^^^^
LL |     d: i32,
   |     ^^^^^^^
LL |     e: i32,
   |     ^^^^^^^
LL |     f: i32,
   |     ^^^^^^^

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0061`.
```

Personally, I prefer pointing at the whole head since the missing paren in the multi-line case looks off-balance.

Reply:
It does a bit, but it makes it less busy (fewer lines going around) and works much better when used by VSCode (overlapping spans look terrible there).